### PR TITLE
Add how-to on custom themes

### PR DESCRIPTION
**Pull request by glrf**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/custom-theme.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+USER 1001:0
+----
+
+
+
+== Deploying Custom Theme
+
+With the theme accessible as a Docker image we can deploy it using an init container.
+In combination with an emptyDir that's shared with the Keycloak container, we copy the theme over to the right place where Keycloak will pick it up automatically.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/custom-theme.adoc:35]:
```suggestion
== Deploying custom theme

With the theme accessible in a container image we can deploy it using an init container.
In combination with an emptyDir volume that's shared with the Keycloak container, we copy the theme over to the right place where Keycloak will pick it up automatically.
```

File: docs/modules/ROOT/partials/nav.adoc [hunk 2]
```diff
@@ -10,6 +10,7 @@
 * xref:how-tos/db-tls.adoc[Encrypt database connection]
 * xref:how-tos/keycloak-tls.adoc[Setup a TLS certificate for Keycloak]
 * xref:how-tos/configure-ingress.adoc[Configure Keycloak ingress]
+* xref:how-tos/custom-theme.adoc[Configure Custom Theme]
```
Comment (ccremer) [inline docs/modules/ROOT/partials/nav.adoc:13]:
```suggestion
* xref:how-tos/custom-theme.adoc[Configure custom theme]
```

File: docs/modules/ROOT/pages/how-tos/custom-theme.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
+= Configure Custom Theme
+
+Keycloak provides theme support for web pages and emails.
+This allows customizing the look and feel of end-user facing pages so they can be integrated with your applications.
+
+Our approach to configure a custom theme in Kubernetes is to bundle the theme in a Docker container and copy it to the correct location as an init container.
```
[the remark this answers is not in the document]
Comment (glrf) [reply docs/modules/ROOT/pages/how-tos/custom-theme.adoc:6]: You corrected this a number of times and I really don't have strong feelings about this, but proper English titles should be capitalized..

[the remark this answers is not in the document]
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/custom-theme.adoc:6]: Where is that coming from?
Thing is, in Project Syn and other various component documentation we only capitalized the first word. So we should stay consistent

[the remark this answers is not in the document]
Comment (glrf) [reply docs/modules/ROOT/pages/how-tos/custom-theme.adoc:6]: Fair enough, but pretty much all English guidelines capitalize titles:

* [APA](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case)
* [AP](https://www.apstylebook.com/)
* [NY Times](https://www.nytimes.com/2018/03/22/insider/new-york-times-stylebook.html)
* [Wikipedia](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Titles)
* A lot more

Here is a good overview page: https://capitalizemytitle.com/

If we decide to not capitalize titles for Project Syn that's fine. But it as a weird design decision.

[the remark this answers is not in the document]
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/custom-theme.adoc:6]: I personally have no hard feelings and would have also capitalized this, but :shrug:

**Review by ccremer — APPROVED** [review]

**Pull request merged**